lib/src/zap/zap.c

```diff
@@ -1011,7 +1011,7 @@ static void init_atfork(void)
 
 	for (i = 0; i < zap_event_workers; i++) {
 		char thread_name[16];
-		(void)snprintf(thread_name, 16, "zap:worker:%d", i);
+		(void)snprintf(thread_name, 16, "zap:wkr:%hd", (short)i);
```

Comment:
@tom95858 I couldn't limit the space of int in the format (unlike float or string). So, I shorten the name "worker" -> "wkr" and specifying "%hd" (short) made the warning went away. I think the short int should be OK .. as the number of zap worker should be below 1,000.